client/ayon_royalrender/rr_root/render_apps/scripts/ayon_inject_envvar.py

```diff
@@ -146,8 +151,9 @@ def _get_job_environments(self):
         env_list = job.customData_Str("rrEnvList")
         envs = {}
         for env in env_list.split("~~~"):
-            key, value = env.split("=")
-            envs[key] = value
+            if "=" in env:
+                key, value = env.split("=")
```

Comment:
We may want to split at most once for if the value contains `=` too? or?
```suggestion
                key, value = env.split("=", 1)
```